misc/scripts/utils/logger.ts

```diff
@@ -0,0 +1,9 @@
+import chalk from 'chalk'
+
+const logger = {
+  info: (msg: string) => console.info(`${chalk.green('[INFO]')} ${msg}`),
+  warning: (msg: string) => console.info(`${chalk.yellow('[WARNING]')} ${msg}`),
+  error: (msg: string, err?: Error | string) => console.info(`${chalk.red('[ERROR]')} ${msg}`, err)
```

Comment:
Can you convert this chalk to clc (from 'cli-color'). This way we'll be able to just remove chalk from our dependencies.